lib/events/proxy.js

```diff
@@ -51,37 +51,40 @@ const createProxyHandler = () => async (req, res) => {
     )
   }
 
-  const signature = req.get('x-etvas-signature')
-  if (!signature) {
-    return res
-      .status(400)
-      .json({ error: 'Invalid request: missing signature header' })
-  }
+  const { suppressSignatureCheck } = config.get('debug') || {}
 
   const { name, payload, timestamp } = req.body
   // Always have a name
   if (!name) {
     return res.status(500).json({ error: 'Empty name' })
   }
 
-  const now = Date.now()
-  const oneMinute = 60000
-  const timeLapse = config.get('eventTimeLapse') || oneMinute
-  if (
-    isNaN(timestamp) ||
-    typeof timestamp !== 'number' ||
-    timestamp <= 0 ||
-    Math.abs(now - timestamp) > timeLapse
-  ) {
-    return res.status(400).json({ error: 'Event timestamp invalid' })
-  }
+  if (suppressSignatureCheck !== true || !process.env.ETVAS_SDK_DEBUG) {
```

Comment:
extract everything in the if condition to a method

Reply:
Maybe it would make sense to have a method that inspects the `debug` config and returns the annotated debug options (with respect to process.env variable).

Something like:
```
const { suppressSignatureCheck } = processDebugConfig()
if (supressSignatureCheck) {
...
```